osquery_rules/osquery_mac_application_firewall.py

```diff
@@ -1,20 +1,22 @@
+from panther_base_helpers import deep_get
+
 QUERIES = {'pack_incident-response_alf', 'pack/mac-cis/ApplicationFirewall'}
 
 
 def rule(event):
-    if event['name'] not in QUERIES:
+    if event.get('name') not in QUERIES:
         return False
 
-    if event['action'] != 'added':
+    if event.get('action') != 'added':
         return False
 
     return (
         # 0 If the firewall is disabled
         # 1 If the firewall is enabled with exceptions
         # 2 If the firewall is configured to block all incoming connections
-        int(event['columns'].get('global_state')) == 0 or
+        int(deep_get(event, 'columns', 'global_state')) == 0 or
         # Stealth mode is a best practice to avoid responding to unsolicted probes
-        int(event['columns'].get('stealth_enabled')) == 0)
+        int(deep_get(event, 'columns', 'stealth_enabled')) == 0)
```

Comment:
Ideally, this should have `try/except` statements around it

Reply:
(which would require us to move it out of the return statement)

Reply:
Is there a specific exception / scenario that you have in mind?

Reply:
Was thinking of a situation where a `None` would yield an int with a default value (0) but that's not relevant as `int(None)` yields a `TypeError`